docs/guides/lint.md

```diff
@@ -3,75 +3,229 @@ redirectFrom:
+The `lint` and `bundle` commands use this section to control various options.
 
-Modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the `lint` or `bundle` commands.
+The `lint` configuration section consists of several sub-sections. The following code block shows an example `lint` section. Sub-sections are described further in the text.
 
-:::attention
+```yaml
+lint:
+  plugins:
+    - './local-plugin.js'
+  extends:
+    - recommended
+  resolve:
+    http:
+      headers:
+        - matches: https://api.example.com/v2/**
+          name: X-API-KEY
+          value: <direct value>
+          envVariable: <name of env variable to be used as value>
+        - matches: https://example.com/*/test.yaml
+          name: Authorization
+          value: <direct value>
+          envVariable: <name of env variable to be used as value>
+  rules:
+    no-sibling-refs:
+      severity: error
+    boolean-parameter-prefixes:
+      severity: error
+      prefixes: ['should', 'is', 'has']
+  ...
+```
 
-When using our hosted Workflows product, the `.redocly.yaml` file must be in the root of the repository.
+## Sub-sections
 
-:::
+### Plugins
 
-Read about the [`--config` option](../commands/index.md) to use other file names or locations.
+Use this section to import local plugins. If you don't have any custom plugins, omit this section.
 
-From a high-level, there are a few sub-sections.
+* **type**: `array of strings`
 
-```yaml
+:::warning
+
+You don't need to import built-in plugins and rules.
+
+Community plugins are not supported.
+
+:::
+
+#### Examples
+
+```yaml Import a single plugin
 lint:
   plugins:
-    # This section is where you can import local plugins.
-    # We don't support community plugins.
-    # You don't need to import our built-in plugins and rules.
-    # Omit this section if you don't have custom plugins.
     - './local-plugin.js'
+```
+
+```yaml Import multiple plugins
+lint:
+  plugins:
+    - ['./local-plugin.js', './another-local-plugin.js']
+```
 
+### Extends
+
+Use this section to choose the base configuration for further extension or adding your own plugins. You may override specific settings in the subsequent sections.
+
+* **type**: `array of strings`
+* **default**: `recommended`
+* **possible values**: `minimal`, `recommended`, `all`
+
+Possible values specified above are related to built-in rules only. Custom plugins can contain additional configurations, so it can be as follows:
+
+```yaml
+lint:
   extends:
-    # This section is where you choose the base configurations.
-    # You may override specific settings in the subsequent sections.
-    - recommended # This is the default (and built in) configuration. If it is too strict, try `minimal`.
+    - recommended
+    - my-custom-super-ruleset
+```
+
+Find more information in the [Configs in plugins](../resources/custom-rules/#configs-in-plugins) section.
```

Comment:
Not to self -- we may need to adjust this to a friendly format for the link checker.